recipes/utf8proc/all/conanfile.py

```diff
@@ -11,7 +12,7 @@ class Utf8ProcConan(ConanFile):
     license = "MIT expat"
     exports_sources = "CMakeLists.txt"
     generators = "cmake"
-    settings = "os", "compiler", "arch", "build_type"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
well, I don't see any reason to constantly re-order that list, actually, order doesn't matter here at all

Reply:
OK. I'm going to stop re-ordering settings.